core/async_job/queue.ex

```diff
@@ -125,28 +118,70 @@ defmodule AntikytheraCore.AsyncJob.Queue do
     end
   end
 
+  defp move_job_running_too_long(q, nil = _target_job_info, _now_millis, _threshold_time), do: q
```

Comment:
Same as https://github.com/access-company/antikythera/pull/51#discussion_r300550170.
```suggestion
  defp move_job_running_too_long(q, nil, _now_millis, _threshold_time), do: q
```

Reply:
I fixed it https://github.com/access-company/antikythera/pull/51/commits/43b95be6693fa8736bf6caf8bd391781a4ccc141